tests/nightly-test.sh

```diff
@@ -0,0 +1,35 @@
+echo "Upload logs"
```

Comment:
i would just call this file logging-tool.sh or something... it is only packaging stuff up, not really a "nightly-test"